### PR TITLE
net-irc/weechat: add 4.3.6, 4.4.1

### PR DESCRIPTION
**Pull request by sevz17**
---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Review by eli-schwartz — CHANGES_REQUESTED** [review]
```
>>> Preparing source in /var/tmp/portage/net-irc/weechat-4.4.1/work/weechat-4.4.1 ...
 * Source directory (CMAKE_USE_DIR): "/var/tmp/portage/net-irc/weechat-4.4.1/work/weechat-4.4.1"
 * Build directory  (BUILD_DIR):     "/var/tmp/portage/net-irc/weechat-4.4.1/work/weechat-4.4.1_build"
 * Applying weechat-3.3-cmake_lua_version.patch ...
patching file cmake/FindLua.cmake
Hunk #1 FAILED at 35.
1 out of 1 hunk FAILED -- saving rejects to file cmake/FindLua.cmake.rej                                                                                                                                    [ !! ]
 * ERROR: net-irc/weechat-4.4.1::gentoo failed (prepare phase):
 *   patch -p1  failed with /var/tmp/portage/net-irc/weechat-4.4.1/files/weechat-3.3-cmake_lua_version.patch
```

**Comment by eli-schwartz** [comment]
See https://github.com/weechat/weechat/pull/2174

I guess this is actually what we want anyways, right? :)

File: net-irc/weechat/weechat-4.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+LICENSE="GPL-3+"
+SLOT="0/${PV}"
+
+NETWORKS="+irc"
+PLUGINS="+alias +buflist +charset +exec +fifo +fset +logger +relay +scripts +spell +trigger +typing +xfer"
+# dev-lang/v8 was dropped from Gentoo so we can't enable javascript support
+# dev-lang/php eclass support is lacking, php plugins don't work. bug #705702
+SCRIPT_LANGS="guile lua +perl +python ruby tcl"
+LANGS=" cs de es fr it ja pl ru"
```
Comment (xen0n) [inline net-irc/weechat/weechat-4.4.1.ebuild:34]: According to the 4.4.1 source:

```suggestion
LANGS=" cs de es fr hu it ja pl pt_BR pt ru sr tr"
```

File: net-irc/weechat/weechat-4.4.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+BDEPEND+="
+	virtual/pkgconfig
+	doc? ( >=dev-ruby/asciidoctor-1.5.4 )
+	man? ( >=dev-ruby/asciidoctor-1.5.4 )
+	nls? ( >=sys-devel/gettext-0.15 )
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.3-cmake_lua_version.patch
```
Comment (xen0n) [inline net-irc/weechat/weechat-4.4.1.ebuild:89]: This is no longer needed for 4.4.1.

**Comment by sevz17** [comment]
I'm sorry for the very long delay.

And thank you @xen0n for the review.

**Comment by eli-schwartz** [comment]
Upgrading to 4.4.1 works fine for me. I did try to /upgrade and it crashed when it tried to reload, but I could not reproduce it when I tried reproducing the upgrade scenario...

**Pull request closed without merging**